src/app/helptext/vm/vm-wizard/vm-wizard.ts

```diff
@@ -78,6 +78,7 @@ volsize_validation: [Validators.required, Validators.min(1)],
 pool_detach_warning_paraText: T("Select zvol"),
 
 datastore_tooltip: T('Choose a pool or dataset for the new zvol.'),
+datastore_placeholder: T('Select zvol'),
```

Comment:
This is not asking for a pool (copy/paste from old UI) or even really a dataset.  It's just a location.  Let's just ask for that.
```suggestion
datastore_placeholder: T('Zvol Location'),
```